core/src/main/java/org/infinispan/cache/impl/DecoratedCache.java

```diff
@@ -423,6 +425,12 @@ public boolean containsKey(Object key) {
    }
 
    @Override
+   public boolean containsValue(Object value) {
+      Objects.nonNull(value);
+      return values().stream().filter(value::equals).findFirst().isPresent();
```

Comment:
Thanks! I actually thought about this while on vacation and totally forgot to fix it when I got back :+1: